charts/stable/photoprism/Chart.yaml

```
@@ -3,7 +3,7 @@ apiVersion: v2
 name: photoprism
 version: 19.1.8
```

```suggestion
version: 19.1.9
```

Need to bump version also.